### DISCUSSION TITLE
Statement timeout. How do you increase the timeout value?

### DISCUSSION THREAD

**Discussion by jaymu53** [supabase/supabase#12887] [Questions]
My query times out when I am querying from the javascript library. 

Looking at this page in the documentation:
https://supabase.com/docs/guides/database/timeouts

I can't execute the following query because of not being a superuser:
alter role authenticated set statement_timeout = '15s';

And where do you need to run this query, since I am making the call from javascript:
set statement_timeout to 60000;

**Comment by GaryAustin1** [comment]
This says contacting support is only way right now.  https://github.com/supabase/supabase/discussions/9314#discussioncomment-4352758
